internal/stacks/stackplan/planned_change.go

```diff
@@ -390,13 +392,20 @@ func (pc *PlannedChangeResourceInstancePlanned) ChangeDescription() (*terraform1
 		}
 	}
 
+	key := pc.ChangeSrc.Addr.Resource.Key.Value()
```

Comment:
Looks like this is producing a null pointer as well - possibly the `Key` part is null when there resource isn't instanced?